src/accordion.js

```diff
@@ -69,6 +70,15 @@ export default class Accordion {
 		let accordionLinks = accordionArea.querySelectorAll( '.accordion-header' );
 		let accordionContent = accordionArea.querySelectorAll( '.accordion-content' );
 
+		// Handle keydown event to move between accordion items
+		accordionArea.addEventListener( 'keydown', () => {
```

Comment:
@AngelMerinoB missing the event parameter, causing a bug on FF

Reply:
Hello @kdo ,
I already commited the changes, it works now on FF. The focus outline is not appearing however (maybe that is more related to CSS) but you can test it using the enter key. I also deleted the extra "eslint-disable" lines. 

Sorry about the closed and reopened PR action a few minutes ago. If there is anything else to modify I'd be glad to help.